### PR TITLE
optimize images by replacing img from StaticImage

### LINKED ISSUE #3017: [Performance] Image Optimization <img> to <StaticImages> in the Meshery page
**Current Behavior**

The .png and .jpg images cause a lot of performance issues
**Desired Situation**

Change  to for image optimization in the handbook section. eg https://github.com/layer5io/layer5/pull/2597 [StaticImagePlugin](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/)
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by avikt18**
Signed-off-by: avikt <avikumrawat18@gmail.com>

**Description**
Replaced the img tags with gatsby's component StaticImage for improving performance.  

This PR fixes #3017 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
It's great to see PRs on this topic coming through. 

@Udit-takkar, it'd be good for you to give this a review.

**Comment by l5io** [comment]
🚀 Preview for commit 97096a0c038fcad869164b0e5138c434742d6462 at: https://63bfb082423d1e20959f2f42--layer5.netlify.app

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
Screenshot 2023-01-12 at 2 44 38 PM

@avikt18  I asked this question on  official discord of gatsby few days ago. so for images other than svg we can use StaticImage which would improve some performance but prefer using img tag for svg images

**Comment by avikt18** [comment]
Yesss @Udit-takkar I read this in the issue #3017 as said by @Jonathan-Foo :
"Yes you have to replace img elements you find with Gatsby's StaticImage component, but don't do it if the image source is an .svg, because it will lower the quality. Yep, you are in the right folder."

File: src/sections/Docker-Meshery/docker-extension-CTA.js [hunk 1]
```diff
@@ -47,7 +49,7 @@ const DockerExtensionCTA = () => {
     <DockerExtensionCTAWrapper>
       <Container>
         <div className="docker-callout">
-          <img src={Dockerimage} alt="Docker and Meshery" />
+          <StaticImage src={Dockerimage} alt="Docker and Meshery" />
```
Comment (Udit-takkar) [inline src/sections/Docker-Meshery/docker-extension-CTA.js:52]: @avikt18 yeah so there are few svg images for which you used StaticImage so if you can change this we are good to merge

Comment (avikt18) [reply src/sections/Docker-Meshery/docker-extension-CTA.js:52]: Oops!! made a mistake :sweat_smile:. On it :+1:

**Comment by l5io** [comment]
🚀 Preview for commit bab31c6a84815545ebcc2968f803fa4ce2450fc5 at: https://63c1c80ab8117e00a1a6a31a--layer5.netlify.app

**Comment by eeshaanSA** [comment]
@avikt18 lets disicuss this in today's websites call. :smile:

**Comment by leecalcote** [comment]
Look good now, @Udit-takkar?

**Comment by Udit-takkar** [comment]
@leecalcote  yeah

**Pull request merged**